Tests/MongoSwiftTests/CrudTests.swift

```diff
@@ -23,16 +23,16 @@ final class CrudTests: MongoSwiftTestCase {
     override class func tearDown() {
         super.tearDown()
         do {
-            try MongoClient().db("crudTests").drop()
+            try MongoClient().db(testDatabase).drop()
```

Comment:
so I guess we don't have a `self` here to take the type of. can we at least prefix the property with the class name so we know where this came from? i.e. `CrudTests.testDatabase`

Reply:
You actually can use `self` in class contexts, and it refers to the type (i.e. `CrudTests`). It's also nice to use for dynamic dispatch in case a subclass overrides any static properties/methods.

Added.

Reply:
oh right, I think we talked about that last week with the `NSNull` init :) sounds good to me.